oauth2client/client.py

```diff
@@ -1222,6 +1227,31 @@ def create_scoped(self, scopes):
         """
         return self
 
+    @classmethod
+    def from_json(cls, s):
+        from oauth2client.service_account import _ServiceAccountCredentials
+        s = _from_bytes(s)
```

Comment:
This one line isn't worth reassigning one of your parameters. How about just `data = json.loads(_from_bytes(s))` for the one line here?